build.sbt

```diff
@@ -259,169 +234,320 @@ lazy val coreJVMTests = Project(id = "coreJVMTests", base = file("enumeratum-cor
   aggregateProject("play-json", enumeratumPlayJsonJs, enumeratumPlayJsonJvm)
+
 lazy val enumeratumPlayJson = crossProject(JSPlatform, JVMPlatform)
   .crossType(CrossType.Pure)
   .in(file("enumeratum-play-json"))
-  .settings(commonWithPublishSettings: _*)
-  .settings(testSettings: _*)
-  .jsSettings(jsTestSettings: _*)
+  .settings(commonWithPublishSettings)
+  .settings(testSettings)
+  .jsSettings(jsTestSettings)
   .settings(
-    name               := "enumeratum-play-json",
-    version            := "1.7.1-SNAPSHOT",
-    crossScalaVersions := Seq(scala_2_12Version, scala_2_13Version),
+    name    := "enumeratum-play-json",
+    version := Versions.Core.head,
+    crossScalaVersions := Seq(
+      scala_2_12Version,
+      scala_2_13Version,
+      scala_3Version
+    ),
+    libraryDependencies ++= Seq(
+      "com.typesafe.play" %%% "play-json" % thePlayJsonVersion(scalaVersion.value),
+      scalaXmlTest.value
+    ),
     libraryDependencies ++= {
-      Seq(
-        "com.typesafe.play" %%% "play-json"       % thePlayJsonVersion(scalaVersion.value),
-        "com.beachape"      %%% "enumeratum"      % Versions.Core.stable,
-        "com.beachape"      %%% "enumeratum-test" % Versions.Core.stable % Test
-      )
+      if (useLocalVersion) {
+        Seq.empty
+      } else {
+        Seq(
+          "com.beachape" %% "enumeratum"      % Versions.Core.stable,
+          "com.beachape" %% "enumeratum-test" % Versions.Core.stable % Test
+        )
+      }
     }
   )
-lazy val enumeratumPlayJsonJs  = enumeratumPlayJson.js
+
+lazy val enumeratumPlayJsonJs = enumeratumPlayJson.js
+  .configure(configureWithLocal(coreJS, "compile->compile;test->test"))
+
 lazy val enumeratumPlayJsonJvm = enumeratumPlayJson.jvm
+  .configure(configureWithLocal(coreJVM, "compile->compile;test->test"))
 
+// Play
 lazy val enumeratumPlay = Project(id = "enumeratum-play", base = file("enumeratum-play"))
-  .settings(commonWithPublishSettings: _*)
-  .settings(testSettings: _*)
+  .settings(commonWithPublishSettings)
+  .settings(testSettings)
   .settings(
-    version            := "1.7.1-SNAPSHOT",
-    crossScalaVersions := Seq(scala_2_12Version, scala_2_13Version),
-    libraryDependencies ++=
-      Seq(
-        "com.typesafe.play" %% "play"            % thePlayVersion(scalaVersion.value),
-        "com.beachape"      %% "enumeratum"      % Versions.Core.stable,
-        "com.beachape"      %% "enumeratum-test" % Versions.Core.stable % Test,
-        scalaTestPlay(scalaVersion.value)
-      )
+    version            := Versions.Core.head,
+    crossScalaVersions := scalaVersionsAll.filter(_ != scala_2_11Version),
+    libraryDependencies += {
+      val dep = ("com.typesafe.play" %% "play" % "2.8.0").exclude("org.scala-lang.modules", "*")
+
+      if (scalaBinaryVersion.value == "3") {
+        dep
+          .exclude("org.scala-lang.modules", "*")
+          .exclude("com.typesafe.play", "play-json_2.13")
+          .cross(CrossVersion.for3Use2_13)
```

Comment:
TIL `CrossVersion.for3use2_13` https://www.scala-lang.org/blog/2021/04/08/scala-3-in-sbt.html#using-scala-213-libraries-in-scala-3